### DISCUSSION TITLE
sensor expired status

### DISCUSSION THREAD

**Discussion by mwarrildt** [NightscoutFoundation/xDrip#3492] [Bug reports]
I'm using a Dexcom G7 sensor with xDrip+ as the receiver on a Samsung Galaxy S21 w/Android 14.

Until today, I've manually stopped the sensor before expiration.  Today I let it expire.  When I went through the process of starting a new sensor. There was no 'Stop Sensor' option.  I removed the old sensor, unpaired the Dx* Bluetooth device, entered the new Transmitter Id and opened the System Status Page.  When prompted I accepted the pairing request for the new transmitter. The sensor and xDrip+ have connected several times but the sensor Status remains 'Sensor Expired'.  There was no 'Start Sensor' option.  

How to get xDrip+ to recognize the new sensor is not expired?
Thanks
Mark

![image](https://github.com/NightscoutFoundation/xDrip/assets/48562773/8effb7d0-37de-4f84-85fa-e39ef4e2b60e)

![image](https://github.com/NightscoutFoundation/xDrip/assets/48562773/abe69d9f-aca9-4aa0-87e7-3cb7432cf08f)

**Comment by Navid200** [comment]
You have blacked out the version number on the status page.  I don't know why.
I cannot help or offer any advice without seeing that.

**Comment by mwarrildt** [comment]
sorry.
I'll also add it's now about 2 hours since insertion
[image: image.png]

**Reply by Navid200** [reply]
Please don't reply to an email.  Attachments will not show.  Also, when you reply to an email, the thread gets messed up.
Please log into GitHub and go to the discussions.  You can see that no screenshot is visible other than the original you had posted.

**Comment by mwarrildt** [comment]
![Screenshot_20240528_153618_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/48562773/eb03c89f-a37a-4ed8-99ad-f3dcb0838016)

If you don't want replies to your email, maybe not include the note at the bottom that says one can reply.

I do appreciate your taking time to help me.

**Reply by Navid200** [reply]
We are not GitHub.  We use GitHub.  You receive the email from GitHub, not from us.

The status page shows that you are connected to your old device.
You need to connect to the new one.
Put your old G7 devices inside a metallic box or the microwave.

**Comment by mwarrildt** [comment]
My apologies. I only meant to note I thought I was following directions
sent to me.

Thank you.

**Reply by Navid200** [reply]
Please put the old device really really far, or inside a metallic box, or in a microwave.
Then, delete the Bluetooth name from the list of previously connected devices on your phone.
Then, wait for a pair request.

Please let me know if it works or not.

**Comment by mwarrildt** [comment]
I normally do use the microwave but my grandson is here, life is a little chaotic and I forgot. But that was only part of it.  I disconnected entirely from xDrip, installed the Dexcom app and, miles from home, the Dexcom app wouldn't connect either. Could be the sensor/transmitter is faulty - so I'll send that sensor back to Dexcom for review.  I deleted Dexcom, installed a new sensor and xDrip connected cleanly.  

Sorry to have troubled you.

**Reply by Navid200** [reply]
I'm glad it is working
